lib/node_runner.go

```diff
@@ -367,27 +290,21 @@ func (nr *NodeRunner) handleBallotMessage(message sebaknetwork.Message) (err err
 		Log:            nr.Log(),
 		VotingHole:     VotingNOTYET,
 	}
-	err = sebakcommon.RunChecker(baseChecker, nr.handleBaseBallotCheckerDeferFunc)
+	err = sebakcommon.RunChecker(baseChecker, nr.handleMessageCheckerDeferFunc)
 	if err != nil {
 		if _, ok := err.(sebakcommon.CheckerErrorStop); !ok {
-			nr.log.Debug("failed to handle ballot", "error", err, "state", "base")
 			return
 		}
-		nr.log.Debug("stop handling ballot", "reason", err, "state", "base")
```

Comment:
Why are those removed ? And why were they `Debug` to begin with ?

Reply:
@Geod24 Oops, it's mistakes including `Debug`. I'll recover them.